01week/rockPaperScissors.js

```diff
@@ -7,12 +7,29 @@ const rl = readline.createInterface({
   output: process.stdout
 });
 
-
 function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
-
-}
+  const move1 = hand1.trim();
+  const move2 = hand2.trim();
+  const move1a = move1.toLowerCase();
+  const move2a = move2.toLowerCase();
+  if (move1a && move2a === 'rock' || 'scissors' || 'paper'){
```

Comment:
This will run with any truthy value, try running with 'dragon' and 5, you'll still be evaluated for the win. What's a better way to check if it's valid?